src/Illuminate/Validation/Validator.php

```diff
@@ -688,6 +688,29 @@ protected function validateRequiredIf($attribute, $value, $parameters)
     }
 
     /**
+     * Validate that an attribute exists when another attribute does not have a given value.
+     *
+     * @param  string  $attribute
+     * @param  mixed   $value
```

Comment:
Sorry, I just copied it over from the method above, hadn't realised it was aligned.